beagle/src/main/kotlin/br/com/zup/beagle/android/view/BeagleActivity.kt

```diff
@@ -333,25 +191,28 @@ abstract class BeagleActivity : AppCompatActivity() {
     fun hasServerDrivenScreen(): Boolean = supportFragmentManager.backStackEntryCount > 0
 
     internal fun navigateTo(requestData: RequestData, screen: Screen?) {
-        fetch(requestData, screen?.toComponent())
+        fetch(requestData, screen)
     }
 
-    private fun fetch(requestData: RequestData, screenComponent: ServerDrivenComponent? = null) {
-        val liveData = screenViewModel.fetchComponent(requestData, screenComponent)
+    private fun fetch(requestData: RequestData, Screen: ServerDrivenComponent? = null) {
```

Comment:
<table data-meta="generated_by_android">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-android/blob/9582f8f47966095c690ec5e84de44f546f4259af/beagle/src/main/kotlin/br/com/zup/beagle/android/view/BeagleActivity.kt#L197"></span>Function parameter names should match the pattern: [a-z][A-Za-z0-9]*</td>
    </tr>
  </tbody>
</table>